resources/dist/linux/share/metainfo/org.moneymanagerex.MMEX.metainfo.xml.in

```diff
@@ -290,37 +290,37 @@
     <screenshot type="default">
       <caption>Dashboard</caption>
       <caption xml:lang="hu">Irányítópult</caption>
-      <image>https://user-images.githubusercontent.com/17465872/233927369-03eabcaa-0e29-4730-8be0-462c3e1c9a01.png</image>
+      <image>https://github.com/moneymanagerex/moneymanagerex/assets/17465872/01cc2202-9d5a-4bba-a816-5868512196cb</image>
```

Comment:
Does the following link need to be used to work:

https://private-user-images.githubusercontent.com/17465872/316453195-01cc2202-9d5a-4bba-a816-5868512196cb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE0ODI4NTQsIm5iZiI6MTcxMTQ4MjU1NCwicGF0aCI6Ii8xNzQ2NTg3Mi8zMTY0NTMxOTUtMDFjYzIyMDItOWQ1YS00YmJhLWE4MTYtNTg2ODUxMjE5NmNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI2VDE5NDkxNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVmMmFiZWQ1ODEyM2M3NzNkMzhiNDIyNzY4Y2UxZjdlYTI1MzRjODQ1NWFjODU5MmYxZDczY2ExY2IwNGE5M2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8GhqpqEhG1MqO4P-KXEQ8wjK1icBaq8mF6oB06e2cjE